rosdep/base.yaml

```diff
@@ -2783,6 +2783,11 @@ libqglviewer-qt4-dev:
     xenial: [libqglviewer-dev-qt4]
     yakkety: [libqglviewer-dev-qt4]
     zesty: [libqglviewer-dev-qt4]
+libqglviewer-qt5-dev:
+  debian: [libqglviewer-dev-qt5]
+  ubuntu:
+    '*': [libqglviewer-dev-qt5]
+    xenial: [libqglviewer-dev]
```

Comment:
@yotabits what package name does Ubuntu 14.04 Trusty use? We'll need to add a specific key for it as well since we still have rosdistros that support it.

If the package is unavailable in trusty I'm not sure we can use the `'*'` form.